feature-group-definitions/webcodecs.yml

```
@@ -1,3 +1,4 @@
+name: "Web Codecs"
```

Aligning with https://developer.mozilla.org/en-US/docs/Web/API/WebCodecs_API:

```suggestion
name: "WebCodecs"
```